adafruit_led_animation/animation/colorcycle.py

```diff
@@ -38,12 +38,15 @@ class ColorCycle(Animation):
     :param float speed: Animation speed in seconds, e.g. ``0.1``.
     :param colors: A list of colors to cycle through in ``(r, g, b)`` tuple, or ``0x000000`` hex
                    format. Defaults to a rainbow color cycle.
+    :param start_color: An index (from 0) for which color to start from. Default 0 (first color).
     """
 
-    def __init__(self, pixel_object, speed, colors=RAINBOW, name=None):
+    # pylint: disable=too-many-arguments
+    def __init__(self, pixel_object, speed, colors=RAINBOW, start_color=0, name=None):
```

Comment:
Since all the args are not keyword-only, adding the new arg to the end will make any calls upward.compatible.
```suggestion
    def __init__(self, pixel_object, speed, colors=RAINBOW, name=None, start_color=0):
```